docbook/src/docbkx/autoscale-devguide.xml

```diff
@@ -70,6 +70,16 @@
         </legalnotice>
         <revhistory>
             <revision xmlns:raxm="http://docs.rackspace.com/api/metadata">
+                <date>2015-03-17</date>
+                <revdescription>
+                    <itemizedlist spacing="compact">
+                        <listitem>
+                            <para>Updated <link linkend="identityEndpoints">Identity endpoints </link>section. Changed launchConfiguration loadBalancerId attribute type to "String."</para>
```

Comment:
I'm not sure what the issue is, the link works fine. I'm unaware of any line length limits; do you mean run-on sentences? Those are my specialty :-) Please feel free to point out any sentence you think could be better written and I'll give it a try.